src/app/app.component.ts

```diff
@@ -54,6 +54,8 @@ export class AppComponent {
   public test: any = {}
 
   public statusPageClient: any
+
+  private isFullscreen: boolean = false
```

Comment:
I think we can get rid of this guy right?

Reply:
Yes and done!